iceoryx_hoofs/source/posix_wrapper/shared_memory_object/memory_map.cpp

```diff
@@ -46,8 +47,11 @@ cxx::expected<MemoryMap, MemoryMapError> MemoryMapBuilder::create() noexcept
                                   static_cast<int32_t>(m_flags),
                                   m_fileDescriptor,
                                   m_offset)
-                      // NOLINTNEXTLINE(cppcoreguidelines-pro-type-cstyle-cast, performance-no-int-to-ptr)
+
+                      // NOLINTJUSTIFICATION cast required, type of error MAP_FAILED defined by POSIX to be void*
+                      // NOLINTBEGIN(cppcoreguidelines-pro-type-cstyle-cast, performance-no-int-to-ptr)
```

Comment:
Out of curiosity: Why did you replace `NOLINTNEXTLINE`?

Reply:
There was a problem with the line break. But maybe it would not have been needed in the latest version ...
Semantically it does not matter and aesthetically it is a lost cause anyway;-)